CHANGELOG

```diff
@@ -95,7 +95,7 @@
 * Use Library/Caches rather than app support directory (Kornel Lesiński)
 * Check for a modal window being onscreen before trying to put up the Sparkle prompt (Alf Watt)
 * Fixed crashes on 10.7 (Chris Campbell, Ger Teunis)
-* Fixed Sparkle tags parsing (Tamás Lustyik)
+* Fixed Sparkle tags parsing (Tamás Lustyik)
```

Comment:
not sure what's happening here, I did all my editing in the GitHub editor, rather than a fancy text editor.